posts/2024-07-20-rladies-bioconductor-collaboration/index.qmd

```diff
@@ -1,13 +1,14 @@
 ---
-title: "Invitation to participate in the new R-Ladies–Bioconductor collaboration!"
+title: "New R-Ladies–Bioconductor collaboration!"
 description: |
   Announcing a new collaboration between R-Ladies and Bioconductor
 author:
   - name: Janani Ravi
     url: https://github.com/jananiravi
-date: 2024-07-20
+date: 2025-01-19
```

Comment:
Can you update the date to when it's to be published this week/month? And can you add to the top of the blog post text: *Original post date: July 2024. Edited in January 2025 to align with the R-Ladies post.* (so it's not confusing for people who read the original post in July).